components/Cta/CtaExternalLink.tsx

```diff
@@ -23,7 +23,7 @@ export const CtaExternalLink = React.forwardRef<HTMLAnchorElement, CtaExternalLi
       iconPosition = 'right',
       animate,
       iconProps,
-      srText = '(external link)',
+      srText,
```

Comment:
SODA said remove all "(external link)" text since there's no visual equivalent and only for screen reader users, and our links don't open in a different window.

Reply:
Huh. This was added on recommendation from SODA on our Sites platform FWIK. Inspect the source at 
https://amptesting.sites.stanford.edu/

@cjwest should this also be changed on the sites theme?

Reply:
@sherakama Yeah I was wondering about that too. Joey is the engineer testing this site. This is her comment:

> Developers must ensure that links have meaningful text within context.  Although it is recommended to warn users when a new window will open, it is not recommended to include "link is external" for links that open in the same window.  This text should be removed.

Reply:
We never got flagged for this before so might be good to talk to the whole SODA team in case there's a difference of opinions 😂  cc @cjwest

Reply:
@yvonnetangsu and @sherakama,

I think this is a case where there's a difference of opinion on how to apply the standards and what is best for all.
I haven't seen the recommendation about opening in a different window. I'm curious about that. However, you probably want to listen to her. She has a deeper background and more experience than me. (and out-ranks me! 😂 )

Reply:
Thank you @cjwest ! 😂